### PR TITLE
Add Core section to roadmap

### PR DESCRIPTION
**Pull request by jtoar**
This PR adds a "Core" section to the Roadmap to cover things like upgrading to ApolloClient 3.0 and Prisma Migrate. @mojombo I think I pushed this branch to the redwoodjs/redwoodjs.com repo so it should be easier to edit this time around if necessary!

### CODE REVIEW

File: lib/docutron.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ const buildNav = (pages, book, index) => {
 const styles = {
   Accessibility: 1,
   Auth: 3,
+  Core: 3,
```
Comment (mojombo) [inline lib/docutron.js:112]: Can you remove the `Prisma Migrate` further down, that's no longer necessary.

Comment (jtoar) [reply lib/docutron.js:112]: Yes, missed that!

**Comment by jtoar** [comment]
Biffed something while trying to compress the commits--fixing now.

**Comment by jtoar** [comment]
All fixed now. Also updated the "Last updated" italics near the top to today's date.

**Pull request merged**